openmdao/solvers/test/test_nl_gauss_seidel.py

```diff
@@ -34,7 +37,13 @@ def test_sellar_group(self):
         prob.root.nl_solver.options['iprint'] = 1 # so that print_norm is in coverage
 
         prob.setup(check=False)
-        prob.run()
+
+        old_stdout = sys.stdout
```

Comment:
I think the iprint was leftover debug, though it probably doesn't hurt to have it. Kind of pointless without testing the output in some way though.